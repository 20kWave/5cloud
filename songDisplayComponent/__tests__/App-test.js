jest.autoMockOff();

import React from 'react';
import ReactDOM from 'react-dom';
const TestUtils = require('react-dom/test-utils');

const App = require('../client/App');

describe('App', () => {
  it('renders the page', () => {
    const rendered = TestUtils.renderIntoDocument(<App />);
    const images = TestUtils.scryRenderedDOMCompnentsWithTag(rendered, 'img');
    expect(images.length).toEqual(1);
  });
});
