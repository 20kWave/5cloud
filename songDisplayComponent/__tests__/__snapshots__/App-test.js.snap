// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component is rendered 1`] = `
<div>
  <div
    className="nav-bar"
  />
  <div
    className="outer-player-panel"
    id="playbackCenter"
  >
    <div
      className="inner-player-panel"
      style={
        Object {
          "background": "linear-gradient(
                135deg,
                rgb(168, 12, 20) 0%,
                rgb(68, 76, 60) 100%",
        }
      }
    >
      <div
        className="player-head"
      >
        <div
          className="play-button-wrapper"
          onClick={[Function]}
        >
          <div
            className="play-button"
          />
        </div>
        <div
          className="artist-name-container"
        >
          <span
            className="artist-name fit-width-to-contents"
          >
            
          </span>
        </div>
        <div
          className="song-name-container"
        >
          <span
            className="song-name fit-width-to-contents"
          >
            
          </span>
        </div>
        <div
          className="date-posted-container"
        >
          <div
            className="date-posted"
          />
        </div>
        <div
          className="tags-container"
        >
          <div
            className="tags fit-width-to-contents"
          >
            
          </div>
        </div>
      </div>
      <div
        className="album-art"
      >
        <img
          alt=""
          className="album-art"
          src=""
        />
      </div>
      <div
        className="song-player"
      >
        <div
          className="current-playback-timer-container"
        >
          <div
            className="current-playback-timer fit-width-to-contents"
          >
            00
          </div>
        </div>
        <div
          className="total-song-length-container"
        >
          <div
            className="total-song-length"
          >
            00:00
          </div>
        </div>
        <div
          className="waveform-container"
        >
          <img
            alt=""
            className="waveform"
            src="./ChartJpg.jpg"
          />
        </div>
        <div
          className="playback-slider-container"
        >
          <input
            className="playback-slider"
            max={60}
            min="0"
            onChange={[Function]}
            style={
              Object {
                "background": "linear-gradient(
                      90deg, 
                      #f50 NaN%, 
                      #999999 0%)",
              }
            }
            type="range"
            value={0}
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
